### DISCUSSION TITLE
训练配置文件数据增强

### DISCUSSION THREAD

**Discussion by CyanMystery** [open-mmlab/mmocr#1971] [Q&A] [answered]
我在train pipeline配置中增加dict(type='RandomFlip', flip_ratio=0.5),这种或者旋转的配置，然后我使用browse_dataset.py文件查看这个这个文件中的训练图片，为啥不显示增强的效果呢？比如图像随机旋转了

**Comment by gaotongxiao** [comment]
You may paste the running command here. Note that you need to pass the modified config to browse_dataset for the correct visualization.

https://mmocr.readthedocs.io/en/dev-1.x/user_guides/useful_tools.html#visualization-tools

**Reply by CyanMystery** [reply, marked as answer]
谢谢，已经可以了 除了在train pipeline中增加配置以外，还需要在train dataset和train dataloader配置块中增加
